src/libs/Pusher/pusher.js

```diff
@@ -387,6 +383,10 @@ function disconnect() {
  * Disconnect and Re-Connect Pusher
  */
 function reconnect() {
+    if (!socket) {
+        console.debug('[Pusher] Unable to reconnect since Pusher instance does not yet exist.');
```

Comment:
Should we be returning here?

Reply:
Ah yep nice catch. Totally meant to do that here. Thanks!!